core/math/math_funcs.h

```diff
@@ -187,6 +187,19 @@ class Math {
 #endif
 	}
 
+	static _ALWAYS_INLINE_ int32_t division_round_up(int32_t p_num, int32_t p_den) {
+		return (p_num + p_den - 1) / p_den;
+	}
+	static _ALWAYS_INLINE_ uint32_t division_round_up(uint32_t p_num, uint32_t p_den) {
+		return (p_num + p_den - 1) / p_den;
+	}
+	static _ALWAYS_INLINE_ int64_t division_round_up(int32_t p_num, int64_t p_den) {
```

Comment:
Any reason `p_num` is 32 bits?

Reply:
No, not quite sure why because I'm quite certain I had checked

Reply:
Same with the unsigned version below, copy-paste error I'd say

Use `git commit --amend` to fix to avoid creating new commits

Reply:
Did you decide this was intentional? Please elaborate why the nominator should be 32 bits here